azure/template.json

```diff
@@ -267,6 +273,10 @@
                                 "name": "RAILS_SERVE_STATIC_FILES",
                                 "value": "[parameters('serveStaticFiles')]"
                             },
+                            {
+                                "name": "SETTINGS__AUTHENTICATION__SECRET",
+                                "value": "[parameters('settingsAuthenticationSecret')]"
```

Comment:
Is this secret being sent to `manage-courses-backend`? If so, what about naming it so that that's clear?

Reply:
its stated on line 135

Reply:
for consistence I keep it the same as it is on frontend

Reply:
https://github.com/DFE-Digital/manage-courses-frontend/blob/master/azure/template.json#L214

Reply:
https://github.com/DFE-Digital/bat-infrastructure/pull/124

Reply:
please please please can we call it `SETTINGS__MANAGE_BACKEND__SECRET`, just based off of the naming we currently use already in mc-fe?

Reply:
https://trello.com/c/Riw3vaQe/1440-settingsauthenticationsecret-on-fe-and-support-needs-to-be-renamed